osu.Game.Rulesets.Taiko/UI/DrawableTaikoRuleset.cs

```diff
@@ -69,7 +69,12 @@ protected override void Update()
             TimeRange.Value = ComputeTimeRange();
         }
 
-        protected virtual double ComputeTimeRange() => PlayfieldAdjustmentContainer.ComputeTimeRange();
+        protected virtual double ComputeTimeRange()
+        {
+            // Adjust when we're using constant algorithm to not be sluggish.
+            double multiplier = VisualisationMethod == ScrollVisualisationMethod.Overlapping ? 1 : 4 * Beatmap.Difficulty.SliderMultiplier;
```

Comment:
There's a change here, `Beatmap.Difficulty.SliderMultiplier` didn't use to be factored in here when this thing was local to `DrawableTaikoEditorRuleset`. What's the reason for changing that?

Reply:
it's to allow mods that change scroll speed (EZ/HR/DA) to work with this. in mania you can change scroll speed via a setting, whereas in taiko you can't. so i'd like to preserve the ability to control scroll speed while making SVs constant.

a good use case is to use this mod + DA scroll speed as a way to practice high/low scroll speed reading.